### DISCUSSION TITLE
How to avoid trigger the click event in `node-props` when click the Tree component suffix part ?

### DISCUSSION THREAD

**Discussion by nit717** [tusen-ai/naive-ui#3971] [Q&A]
Question
 
When I click the tree suffix part, it will trigger the `suffix click event`(:render-suffix prop) and the `tree click event`(:node-props prop) at the same time, and the question is actually it will auto select the tree item when I click the suffix part, so is there any solution to solve this problem  when I just wanna trigger the suffix clikc event ,not select the tree item ?

<img width="617" alt="image" src="https://user-images.githubusercontent.com/66296731/198967288-4e9dd0b4-9fcc-43c5-8fbd-3d5e57e8196b.png">

**Comment by XieZongChen** [comment]
`e.preventDefault()` ?
